pos_payment_restriction/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+In some situations, one would have user restrictions on payment lines level.
+
+This includes :
+
+* An option to make payment amount readonly
+* An option that adds a helper to disable the 'Validate' button (e.g.:
+  waiting for an automatic payment transaction to be completed)
```

Comment:
@glitchov this part about the validation module is not part of this module anymore, right. It seems to be standard Odoo.